fdbserver/VersionedBTree.actor.cpp

```diff
@@ -1401,25 +1401,22 @@ struct RedwoodMetrics {
 			metrics = {};
 			if (!buildFillPctSketch.isValid() ||
 			    buildFillPctSketch->name() != ("buildFillPct:" + std::to_string(levelCounter))) {
-				buildFillPctSketch = Histogram::getHistogram(LiteralStringRef("buildFillPct"),
-				                                             LiteralStringRef(std::to_string(levelCounter).c_str()),
-				                                             Histogram::Unit::percentage);
-				modifyFillPctSketch = Histogram::getHistogram(LiteralStringRef("modifyFillPct"),
-				                                              LiteralStringRef(std::to_string(levelCounter).c_str()),
-				                                              Histogram::Unit::percentage);
-				buildStoredPctSketch = Histogram::getHistogram(LiteralStringRef("buildStoredPct"),
-				                                               LiteralStringRef(std::to_string(levelCounter).c_str()),
-				                                               Histogram::Unit::percentage);
-				modifyStoredPctSketch = Histogram::getHistogram(LiteralStringRef("modifyStoredPct"),
-				                                                LiteralStringRef(std::to_string(levelCounter).c_str()),
-				                                                Histogram::Unit::percentage);
+				std::string levelCounterStr = std::to_string(levelCounter);
+				buildFillPctSketch = Histogram::getHistogram(
+				    LiteralStringRef("buildFillPct"), StringRef(levelCounterStr), Histogram::Unit::percentage);
```

Comment:
Maybe use `"buildFillPct"_sr` instead? And a few other places as well.

Reply:
There's like 2500 such places. I think it's out of scope for this change (although I personally would like to see LiteralStringRef deprecated)

Reply:
I created #5229 which will do this